lib/facter/apache_version.rb

```diff
@@ -2,11 +2,9 @@
   setcode do
     if Facter::Util::Resolution.which('apachectl')
       apache_version = Facter::Util::Resolution.exec('apachectl -v 2>&1')
-      puts "Matching apachectl '#{apache_version}'"
```

Comment:
Wouldn't it be a better solution to change this to:
`Puppet.debug("Matching apachectl '#{apache_version}'")`
or
`Puppet.info("Matching apachectl '#{apache_version}'")`

See here http://www.rubydoc.info/gems/puppet/Puppet/Util/Logging#debug-instance_method